packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -462,6 +474,14 @@ export default class ElectronAppWrapper {
 		this.customProtocolHandler_ ??= handleCustomProtocols(logger);
 	}
 
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	public initializeAutoUpdaterService(logger: LoggerWrapper, initializedShim: any, devMode: boolean, allowPrereleaseUpdates: boolean) {
+		if (!shim.isLinux()) {
```

Comment:
Here please only list the platforms that are supported, which would be isMacOS and isWindows. Joplin works also on BeOS and maybe others, so it's better to explicitly list what we support instead what we don't support